config/sync/core.entity_view_display.node.health_condition.default.yml

```diff
@@ -117,9 +113,19 @@ content:
   field_summary:
     weight: 6
     label: above
-    settings: {  }
+    settings:
```

Comment:
Is this autogenerated config? Looks unimportant being on the default display mode but unclear why it's coming out as part of this work/at this time.

Reply:
Again, I tried this again with the same end result, certainly doesn't look harmful.